feature-examples/python/api_key/list_api_key.py

```diff
@@ -0,0 +1,55 @@
+"""Example of using the Vectara REST API to list the Api Keys."""
+
+import logging
+from typing import Optional
+
+import requests
+
+
+def list_apikeys(
+    customer_id: int,
+    jwt_token: str,
+) -> tuple[Optional[str], bool]:
+    """Retrieves the list of API keys.
+
+    Args:
+        customer_id: Unique customer ID in vectara platform.
+        jwt_token: JWT token to be used for authentication.
+
+    Returns:
+        (message, True) in case of success and returns (status, False) in case of failure.
```

Comment:
shouldn't we return the actual keys if successful? The return type would then be `Union[list[str], str]`, which is not great, but at least would demonstrate the feature. Exceptions would be a nicer way to handle the failure case here. Consider returning a list of API keys on success or raising an exception in the case of failure.

Reply:
Agreed. Done.